src/inboxsdk.d.ts

```diff
@@ -15,18 +15,7 @@ import type { AppLogger } from './platform-implementation-js/lib/logger';
 export * from './platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view';
 
 export const LOADER_VERSION: string;
-export interface LoadScriptOptions {
```

Comment:
We do depend on this interface in our downstream repo at Streak. Instead of removing this entirely, we might add an additional module `remote.d.ts` with this and TSDoc to the effect that:

> This interface only exists in the Manifest V2 (MV2) remote loaded version of the SDK. If and when MV2 support is dropped, this interface will be removed.